fuel-derive/src/deserialize.rs

```diff
@@ -92,36 +87,30 @@ fn deserialize_enum(s: &synstructure::Structure) -> TokenStream2 {
                 }
             });
 
+            if variant.ast().discriminant.is_some() {
```

Comment:
is this native discriminant stable across rust versions?

Reply:
Yep, I used `x if x == { Self::Mint as u64 }`. But I decided to replace it with const. Now it looks like:

```rust
const V1: u64 = { Transaction::Mint as u64 };

match value {
    V1 => { ... }
    _ => { ... }
}
```